Documentation/ceph-cluster-crd.md

```diff
@@ -134,8 +134,9 @@ The following storage selection settings are specific to Ceph and do not apply t
 - `walSizeMB`:  The size in MB of a bluestore write ahead log (WAL). Include quotes around the size.
 - `journalSizeMB`:  The size in MB of a filestore journal. Include quotes around the size.
 - `osdsPerDevice`**: The number of OSDs to create on each device. High performance devices such as NVMe can handle running multiple OSDs. If desired, this can be overridden for each node and each device.
+- `encryptedDevice`**: Encrypt OSD volumes using dmcrypt ("true" or "false").
```

Comment:
Could you add a comment here and in the cluster.yaml that the default is `false`? After that, LGTM